packages/system-tests/assets/sfdx-simple/.vscode/settings.json

```diff
@@ -1,3 +0,0 @@
-{
```

Comment:
Why do we need to delete this ?

Reply:
Thanks for catching this. I did not mean to delete this and will put it back.